### PR TITLE
 Code Review for To Do list structure

### PR DESCRIPTION
**Pull request by luigirazum**
### The following requirements were fulfilled 📌 in this PR:
- ### [✅] Set up a new project with webpack that is based on the webpack exercise you have already completed.
- ### [✅] Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (`<div>` or `<ul>` element). The index.html file must be set as a template using the HTML Webpack Plugin.
- ### [✅] Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
   - ### [✔] description: `string`.
   - ### [✔] completed: `bool`.
   - ### [✔] index: `number`.
- ### [✅] Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- ### [✅] On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
- ### [✅] Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader.
- ### [✅] All your source files (index.html, index.js and style.css) must be located in /src directory and your distribution files will be generated by webpack and served by webpack dev server from /dist folder.

### Luis 🙋‍♂️

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -1 +1,40 @@
...
+        <span class="item">Complete the project</span>
+        <span class="material-symbols-outlined icon">more_vert</span>
+      </li>
+      <li class="btn-sweep">
+        <span>Clear all completed</span>
+        <span class="material-symbols-outlined icon">delete_sweep</span>
+      </li>
+      <!-- <li class="title"><span class="material-symbols-outlined">task_alt</span></li>
+        <li class="title"></li>
+        <li class="title"><span class="material-symbols-outlined">delete</span></li> -->
```
Comment (Owusu-Desmond) [inline dist/index.html]: - [ ] Good job so far :+1: Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @luigirazum , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 1 above)
Comment (luigirazum) [reply dist/index.html]: Thanks for the feedback @Owusu-Desmond 👍.
I've already asked for another CR.
Cheers,
Luis 👋

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @luigirazum, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**